### DISCUSSION TITLE
player-sdfat-i2s - sdfat and esp32-a1s audio kit

### DISCUSSION THREAD

**Discussion by StefanBu1970** [pschatzmann/arduino-audio-tools#1515] [Q&A] [answered]
Hi,
I am trying to adpot the player-sdfat-i2s example, but can not manage to ope SD Card. SD Card is fotmatted FAT32 and has 8MB.
Error code is
[E] AudioSourceSDFAT.h : 78 - sd.begin failed
[I] AudioSourceSDFAT.h : 101 - selectStream SDFAT: 0
[E] AudioSourceSDFAT.h : 111 - Filename is null

I assume, that SDFat has the wrong pinout, but I did not find how to adapt...

**Comment by pschatzmann** [comment, marked as answer]
https://github.com/pschatzmann/arduino-audio-tools/wiki/It's-not-working

Did you manage to read your files outside of my library ?
If not that would be the first thing to do...

**Reply by StefanBu1970** [reply]
Not at all, but I found I used the wrong example. Now trying player-sdfat-audiokit, but currently loops.
Which Board do You use in Arduino IDE?

**Reply by StefanBu1970** [reply]
One You use the right examples, everything works fine.
Thanks a lot for Your quick answer.
Closing thread
